internal/pkg/pkg_test.go

```diff
@@ -129,7 +129,7 @@ kind: Custom
 spec:
   image: nginx:1.2.3`,
 				`
-apiVersion: kpt.dev/v1alpha1
+apiVersion: kpt.dev/v1alpha2
```

Comment:
the schema here needs to change, it's using `setterDefinitions`

Reply:
ahhh... good catch, just saw this comment. Ignore the re-request.

Reply:
fixed now and I checked other places and didn't find any out of date definitions.